src/FileIO.scala

```diff
@@ -0,0 +1,67 @@
+import java.io._
+import java.util
+
+import scala.collection.mutable
+import scala.io.Source
+
+
+/**
+  * Created by joshuakeough on 10/18/16.
+  */
+object FileIO {
+
+  val purchases = mutable.MutableList[Purchase]()
+
+  def main(args: Array[String]): Unit = {
+
+
+    Source.fromFile("purchases.csv").getLines.drop(1).foreach(line => {
+      val Array(customer_id, date, credit_card, cvv, category) = line.split(",").map(_.trim)
+      purchases += Purchase(customer_id, date, credit_card, cvv, category)
+    })
+    var resp = ""
+    while (resp != "Q") {
+      resp = menu match {
+        case "FURNITURE" => sort("furniture")
+        case "ALCOHOL" => sort("alcohol")
+        case "TOILETRIES" => sort("toiletries")
+        case "SHOES" => sort("shoes")
+        case "FOOD" => sort("food")
+        case "JEWELRY" => sort("jewelry")
```

Comment:
What would you do if the file had 500 categories?  Certainly not hard code each.  
How would you match on ANY variable?